examples/integration_demo.py

```
@@ -0,0 +1,353 @@
+              get("hand_side")
+              ...
+
+=== 运行方式 ===
+
+终端 1: 启动一代手桥接
+    wujihandpy-bridge --sn <一代手SN> --pub-rate 100
+
+终端 2: 启动触觉桥接
+    wujihandpy-tactile-bridge --sn <tboard_SN> --pub-rate 30
+
+终端 3: SDK 订阅端（本脚本）
```

Ruff (0.15.6)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)